Xamarin.Forms.Core/Shapes/CompositeTransform.cs

```diff
@@ -0,0 +1,140 @@
+﻿namespace Xamarin.Forms.Shapes
+{
+	public sealed class CompositeTransform : Transform
```

Comment:
can't we reuse the Transform we already have, that's also hardware accelerated in some cases ?